utilities/help.py

```diff
@@ -183,7 +185,7 @@ async def send_group_help(self, group: commands.Group):
         )
         await self.context.send(embed=embed)
 
-    async def send_error_message(self, error):
+    async def send_error_message(self, error: str) -> None:
```

Comment:
How does this work? This function only gets a string of the error, and not the error instance itself?

Reply:
According to the documentation, yes.